### ISSUE TITLE
AppVeyor build hangs

### ISSUE THREAD

**Issue by jagrem** [jugglingnutcase/language-powershell#41]
AppVeyor CI build is hanging on `cinst Atom` while it's waiting for user input and then the build times out.
Possibly add a `-Force` flag to `cinst Atom` in `appveyor.yml`?

I can test once I have a Windows system in front of me again :)

**Comment by vors** [comment]
I think it `-y`:

```
-y, --yes, --confirm
    Confirm all prompts - Chooses affirmative answer instead of prompting.
      Implies --accept-license
```

**Comment by jagrem** [comment]
Ah that sounds more sensible :)

I'll give that a try and create a PR.

**Comment by jugglingnutcase** [comment]
@jagrem, thanks! i noticed this too, but didn't dig into it yet. Mind sending a PR to atom/ci as well when you do that?

**Comment by jugglingnutcase** [comment]
Fixed by #44
